### ISSUE TITLE
unconvert doesn't work in certain scenarios with Go modules

### ISSUE THREAD

**Issue by nmiyake** [mdempsky/unconvert#34]
To repro, perform the following outside of the `$GOPATH`:

```
➜  mkdir test-mod
➜  cd test-mod
➜  go mod init github.com/test/mod
go: creating new go.mod: module github.com/test/mod
➜  echo 'package foo; import _ "gopkg.in/AlecAivazis/survey.v1"; func foo() { v := 5; _ = int(v);  }' > main.go
➜  go build .
➜  unconvert .
/Volumes/git/go/pkg/mod/gopkg.in/!alec!aivazis/survey.v1@v1.7.1/confirm.go:7:2: could not import gopkg.in/AlecAivazis/survey.v1/core (cannot find package "gopkg.in/AlecAivazis/survey.v1/core" in any of:
	/usr/local/go/src/gopkg.in/AlecAivazis/survey.v1/core (from $GOROOT)
	/Volumes/git/go/src/gopkg.in/AlecAivazis/survey.v1/core (from $GOPATH))
/Volumes/git/go/pkg/mod/gopkg.in/!alec!aivazis/survey.v1@v1.7.1/editor.go:10:13: could not import github.com/kballard/go-shellquote (cannot find package "github.com/kballard/go-shellquote" in any of:
	/usr/local/go/src/github.com/kballard/go-shellquote (from $GOROOT)
...
```

This is tracked on the Go side as https://github.com/golang/go/issues/24661.

Using the new `"golang.org/x/tools/go/packages"` API seems to resolve this issue.

**Comment by mdempsky** [comment]
I think this was fixed by #32. Please let me know if I'm mistaken.

**Comment by nmiyake** [comment]
Yes, #32 resolves this, and verified that this now works on master -- thanks!
